src/CalcViewModel/Common/CopyPasteManager.cpp

```diff
@@ -69,7 +69,8 @@ void CopyPasteManager::CopyToClipboard(String ^ stringToCopy)
     Clipboard::SetContent(dataPackage);
 }
 
-IAsyncOperation<String ^> ^ CopyPasteManager::GetStringToPaste(ViewMode mode, CategoryGroupType modeType, NumberBase programmerNumberBase, BitLength bitLengthType)
+IAsyncOperation<
```

Comment:
Seems clang-format is not able to help arrange C++/CX code in the way we'd like to see.
For example, clang-format might have treated `^` as a XOR operator and get confused here.